crypto/evp/digest.c

```diff
@@ -539,30 +539,32 @@ int EVP_MD_CTX_get_params(EVP_MD_CTX *ctx, const OSSL_PARAM params[])
     return 0;
 }
 
-#if !OPENSSL_API_3
```

Comment:
I believe some of the documentation says that EVP_MD_CTX_ctrl is deprecated. If we're going down that route then that should be updated.

Reply:
updated